mailer/views.py

```diff
@@ -29,6 +29,13 @@ def get_queryset(self):
         return Groups.objects.filter(user=self.request.user.id)
 
     def post(self, request, *args, **kwargs):
+
+        if not request.user.is_authenticated:
```

Comment:
authentication is being checked above.
on `line 22`
`permission_classes = [IsAuthenticated]` 
you have to validate if the authenticated user is the same user that is being passed in the json request of the api.

Reply:
ok then i will change it and again make the pull request